sdk/devcenter/azure-developer-devcenter/tsp-location.yaml

```diff
@@ -0,0 +1,4 @@
+directory: specification/devcenter/DevCenter
+commit: 74e93fc9fb00380250110fba78dadfebeb93cbf1
```

Comment:
This is the latest commit on the branch?

Reply:
yes